src/SprykerEco/Zed/UnzerApi/Business/UnzerApiFacadeInterface.php

```diff
@@ -14,6 +14,8 @@ interface UnzerApiFacadeInterface
 {
     /**
      * Specification:
+     * - Requires UnzerApiRequestTransfer::unzerKeypair to be set.
```

Comment:
Please, use quotes for all cases like this `UnzerApiRequestTransfer::unzerKeypair`.